.github/workflows/ci.yml

```
@@ -14,42 +14,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node: ["16", "15", "14", "12", engines]
-        exclude:
-          # On Windows, run tests with only the LTS environments.
-          - os: windows-latest
-            node: engines
-          - os: windows-latest
-            node: "14"
-          # On macOS, run tests with only the LTS environments.
-          - os: macOS-latest
-            node: engines
-          - os: macOS-latest
-            node: "14"
+        node: ["17.3"]
```

Shouldn't we include the currently supported versions of Node.js here?

```suggestion
        node: ["12.20.0", "14.13.0", "16.0.0", "17"]
```

---

The http module loader API changed...

---

can revisit this later.